modules/sql/src/main/resources/db/migration/V015__half.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TYPE half AS ENUM (
+    'A', 'B'
+);
```

Comment:
You mean an enumeration for `Science`, `Daily`, `Nonstandard`? Yes I think so and this tag can be used as the basis for the ordering question above.